### DISCUSSION TITLE
New user can't connect

### DISCUSSION THREAD

**Discussion by AlexTank1664** [strongswan/strongswan#2406] [Q&A]
Hello all!
I have fully functional strongswan on centos, but I can't add new users. 
I added a row in ipsec.secrets 
alex : EAP "password" 
but there is an error in log file 
Aug 15 22:54:01 server charon[225603]: 16[IKE] no EAP key found for hosts '55.55.55.55' - 'alex'

How I can add new users to use vpn?

**Comment by strongX509** [comment]
During the startup of the charon daemon, does the log show that the EAP password has been successfully loaded as in
```
charon: 00[CFG]   loaded EAP secret for alex
```

**Reply by AlexTank1664** [reply]
Yes, there are lines for both new and old users in log
Aug 21 05:23:23 server charon-systemd[74467]:  loaded RSA private key from '/etc/strongswan/ipsec.d/private/server-key.pem'
Aug 21 05:23:23 server charon-systemd[74467]:  loaded EAP secret for admin
Aug 21 05:23:23 server charon-systemd[74467]:  loaded EAP secret for alex
Aug 21 05:23:23 server charon-systemd[74467]:  loaded EAP secret for alex1664
